sql/mob_groups.sql

```diff
@@ -620,7 +620,7 @@ INSERT INTO `mob_groups` VALUES (34,1840,15,'Guimauve',0,32,1250,0,999,80,85,0);
 INSERT INTO `mob_groups` VALUES (35,1348,15,'Fistule',900,0,835,0,0,90,90,0);
 INSERT INTO `mob_groups` VALUES (36,3235,15,'Pustule',0,128,0,0,0,78,88,0);
 INSERT INTO `mob_groups` VALUES (37,2743,15,'Morboling',300,0,1737,0,0,79,80,0);
-INSERT INTO `mob_groups` VALUES (38,742,15,'Clingy_Clare',0,128,479,0,0,0,0,0);
+INSERT INTO `mob_groups` VALUES (38,742,15,'Clingy_Clare',0,128,479,0,0,80,85,0);
```

Comment:
this from captures or a guess? The wiki's don't show me a level and thing usually don't have a 5 level range

Reply:
I made my own capture on retail 3 times. I corrected to 85-85 because I had misread one of the captures I made.

`    [16838946] = {['id']=16838946, ['name']="ClingyClare",             ['polutils_name']="Clingy Clare",                       ['type']="Simple NPC",   ['index']=290, ['level']=85,  ['x']=150.000,  ['y']=17.125,   ['z']=90.000,   ['r']=9,   ['flag']=4389,   ['speed']=40, ['speedsub']=40, ['animation']=3,  ['animationsub']=8,  ['namevis']=0,   ['status']=7,  ['flags']=671,     ['name_prefix']=32,  ['look']="00007E01",                                 ['raw_packet']="0E22B90222F1000122010F0900001643000089410000B4422511000028280003079F0200002C002020000800CFFF070000007E01436C696E6779436C6172650000000000",},
`

` 
 [16838946] = {['id']=16838946, ['name']="ClingyClare",             ['polutils_name']="Clingy Clare",                       ['type']="Simple NPC",   ['index']=290, ['level']=85,  ['x']=150.000,  ['y']=17.125,   ['z']=90.000,   ['r']=8,   ['flag']=47,     ['speed']=40, ['speedsub']=40, ['animation']=1,  ['animationsub']=8,  ['namevis']=0,   ['status']=1,  ['flags']=671,     ['name_prefix']=32,  ['look']="00007E01",                                 ['raw_packet']="0E22140622F1000122010F0800001643000089410000B4422F00000028286401019F0200002C002020000800CFFF070000007E01436C696E6779436C6172650000000000",},
`

`    [16838946] = {['id']=16838946, ['name']="Clingy_Clare",            ['polutils_name']="Clingy Clare",                       ['type']="Equipped NPC", ['index']=290, ['level']=85,  ['x']=150.000,  ['y']=17.125,   ['z']=90.000,   ['r']=236, ['flag']=1697,   ['speed']=40, ['speedsub']=40, ['animation']=1,  ['animationsub']=8,  ['namevis']=0,   ['status']=1,  ['flags']=671,     ['name_prefix']=32,  ['look']="00007E01",                                 ['raw_packet']="0E1CA00222F10001220107EC00001643000089410000B442A106000028286001019F0200002C002020000800CFFF070000007E0100000000",},
`

Reply:
add that the 3rd cap has an underscore the 1st 2 didn't.